src/helpers/axis.jsx

```diff
@@ -7,15 +20,11 @@ type Tick = {
   dy: number,
   textAnchor: string,
   val: number,
-}
+};
 
-export default function getTicks(
-  min: number,
-  max: number,
-  numTicks: number,
-  axisType: string,
-  position: number,
-  context: JubilationContext): Tick[] {
+export default function getTicks({
+  min, max, numTicks, axisType, position, context,
```

Comment:
As noted in #15, we'll need to refactor places where this new method signature is used